### PR TITLE
[Player-4171] Update exoplayer version

### PR DESCRIPTION
**Pull request by SergeyBicarte**
synced with dev branch

### CODE REVIEW

**Comment by pilievOoyala** [comment]
LGTM

File: VRSampleAppKotlin/app/src/main/java/com/ooyala/sample/screen/VideoRecyclerFragment.kt [hunk 1]
```diff
@@ -7,6 +7,7 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import com.ooyala.sample.R
+import com.ooyala.sample.R.id.videoRecyclerView
```
Comment (SZadorozhnyiEpam) [inline VRSampleAppKotlin/app/src/main/java/com/ooyala/sample/screen/VideoRecyclerFragment.kt]: This line is probably unused

File: settings.gradle [hunk 2]
```diff
@@ -67,10 +61,16 @@ include ':HeartbeatSampleApp'
 project(':HeartbeatSampleApp').projectDir = new File(settingsDir, './HeartbeatSampleApp/app')
 
 // Developer
-//include ':ooyalaSkinSDK'
-//project(':ooyalaSkinSDK').projectDir=new File(settingsDir, '../native-skin/sdk/android/ooyalaSkinSDK/')
+//include ':skin'
+//project(':skin').projectDir=new File(settingsDir, '../native-skin/sdk/android/skin/')
```
Comment (SZadorozhnyiEpam) [inline settings.gradle]: codestyle: check spaces before/after `=`

**Review by SZadorozhnyiEpam — COMMENTED** [review]
Left a few comments

File: FreewheelSampleApp/app/build.gradle [hunk 3]
```diff
@@ -30,9 +30,18 @@ task copyTask(type: Copy) {
 
 tasks.copyTask.execute()
 
+project.configurations.all {
+    resolutionStrategy.eachDependency { details ->
+        if (details.requested.group == 'com.android.support'
+                && !details.requested.name.contains('multidex')) {
+            details.useVersion "27.1.1"
```
Comment (SZadorozhnyiEpam) [inline FreewheelSampleApp/app/build.gradle]: version should be extracted as constant. If we update appcompat version this code might downgrade version for other support libraries

Comment (SZadorozhnyiEpam) [inline FreewheelSampleApp/app/build.gradle]: version should be extracted as constant. If we update appcompat version this code might downgrade version for other support libraries

**Review by SZadorozhnyiEpam — COMMENTED** [review]
Also this

File: FreewheelSampleApp/app/build.gradle (hunk 3 above)
Comment (SZadorozhnyiEpam) [reply FreewheelSampleApp/app/build.gradle]: Basically it should be like this
`details.useVersion ${appCompatVersion}`
`implementation "com.android.support:appcompat-v7:${appCompatVersion}"`

File: VRSampleAppKotlin/app/src/main/java/com/ooyala/sample/screen/VideoRecyclerFragment.kt (hunk 1 above)
Comment (SergeyBicarte) [reply VRSampleAppKotlin/app/src/main/java/com/ooyala/sample/screen/VideoRecyclerFragment.kt]: Done

File: settings.gradle (hunk 2 above)
Comment (SergeyBicarte) [reply settings.gradle]: Done

File: FreewheelSampleApp/app/build.gradle (hunk 3 above)
Comment (SergeyBicarte) [reply FreewheelSampleApp/app/build.gradle]: Done

**Pull request merged**